apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverview.tsx

```diff
@@ -729,8 +728,7 @@ export const SignedVerdictOverview: React.FC = () => {
             workingCase.prosecutorAppealDecision ===
               CaseAppealDecision.APPEAL) &&
             user?.role &&
-            isCourtRole(user.role) &&
-            user?.institution?.type !== InstitutionType.HighCourt && (
```

Comment:
One thing about this change is that I'm removing all if statements regarding if a user is a high court (court of appeal) judge from SignedVerdictOverview page because they now won't be using this page but rather a new court of appeal overview page.